templates/default/sv-sentry-cron-log-run.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
In that case, let's create a separate template directory for sentry v8 templates